src/index.css

```diff
@@ -25,3 +25,58 @@ code {
     'Courier New',
     monospace;
 }
+
+table,
+th,
+td {
+  border: 1px solid rgb(131, 131, 131);
+}
+
+.mission-table thead tr {
+  text-align: left;
+}
+
+.table-head {
+  padding: 0.5em 10px;
```

Comment:
I would suggest to move all these into the app.css and delete this index.css in order to have all your stylings into one file which help in code maintenance and reusability.